frontend/rust-lib/flowy-database2/tests/database/sort_test/multi_sort_test.rs

```diff
@@ -5,46 +5,44 @@ use crate::database::sort_test::script::DatabaseSortTest;
 use crate::database::sort_test::script::SortScript::*;
 
 #[tokio::test]
-async fn sort_text_with_checkbox_by_ascending_test() {
+async fn sort_checkbox_and_then_text_by_descending_test() {
   let mut test = DatabaseSortTest::new().await;
-  let text_field = test.get_first_field(FieldType::RichText).clone();
-  let checkbox_field = test.get_first_field(FieldType::Checkbox).clone();
+  let checkbox_field = test.get_first_field(FieldType::Checkbox);
+  let text_field = test.get_first_field(FieldType::RichText);
   let scripts = vec![
-    AssertCellContentOrder {
-      field_id: text_field.id.clone(),
-      orders: vec!["A", "", "C", "DA", "AE", "AE"],
-    },
     AssertCellContentOrder {
       field_id: checkbox_field.id.clone(),
-      orders: vec!["Yes", "Yes", "No", "No", "No", "Yes"],
-    },
-    InsertSort {
-      field: text_field.clone(),
-      condition: SortCondition::Ascending,
+      orders: vec!["Yes", "Yes", "No", "No", "No", "Yes", ""],
     },
     AssertCellContentOrder {
       field_id: text_field.id.clone(),
-      orders: vec!["A", "AE", "AE", "C", "DA", ""],
-    },
-    AssertCellContentOrder {
-      field_id: checkbox_field.id.clone(),
-      orders: vec!["Yes", "No", "Yes", "No", "No", "Yes"],
+      orders: vec!["A", "", "C", "DA", "AE", "AE", "CB"],
     },
-  ];
-  test.run_scripts(scripts).await;
-
-  let scripts = vec![
+    // Insert checkbox sort
     InsertSort {
```

Comment:
I think it would be better to assert the order first that can help the other developers understand how the order changes by inserting a sort.

Reply:
I combined the two scripts into one so the lines above assert the order. Or maybe I misunderstood?

Reply:
I combined the two scripts into one so the lines above assert the order. Or maybe I misunderstood?